src/components/templates/PartyMap/components/RoomAttendance/RoomAttendance.tsx

```diff
@@ -19,7 +18,7 @@ export const RoomAttendance: React.FC<RoomAttendanceProps> = ({
   room,
   maxVisible = DEFAULT_ROOM_ATTENDANCE_LIMIT,
 }) => {
-  const { portalSpaceId } = usePortal({ portal: room });
+  const portalSpaceId = room.spaceId;
```

Comment:
Saw this as I was chasing some loose ends. Saw that `usePortal` really wasn't needed. Introduces an unnecessary query.

Reply:
Not sure if that field was present in the past when the hook was introduced. Confident all the portals have their parent id?

Reply:
It's basically the same code without the use of a hook. This isn't the parent ID. This is the space that the portal will take you. If a portal doesn't have a target space then it's not going to work **at all**